release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1051,61 +1355,371 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
+* Previously, a missing nil check for a route condition could cause the OpenShift CLI (`oc`) to crash when describing a route. A nil check was added and describing a route now works properly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1893645[*BZ#1893645*])
 
-* While running a Pipeline, the log tab of the `TaskRun` resource displayed the string as `undefined` after the command in the output. This was caused due to some edge cases where some internal string operations printed  `undefined` to the log output. This issue has been fixed now, and the pipeline log output does not drop empty lines from the log stream and does not print the string `undefined` any longer. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1915898[*BZ#1915898*])
+* Previously, the OpenShift CLI (`oc`) had a low limit for client throttling, and the requests reaching for API discovery were limited by the client code. The client throttling limit was increased and client-side throttling should now appear less frequently. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1899575[*BZ#1899575*])
 
-* Previously, the *Port* list in the *Add* flow only provided options for exposed ports and did not allow you to specify a custom port. The list has now been replaced by a typeahead select menu, and now it is possible to specify a custom port while creating the application. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1881881[*BZ#1881881*])
+* Previously, support for init containers was lost during changes to the `oc debug` command, and it was not possible to debug init containers. Support for init containers was added to the `oc debug` command, and it is now possible to debug init containers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1909289[*BZ#1909289*])
 
-* Previously, when conditional tasks failed, the completed pipeline runs showed a permanent pending task for each failed conditional task. This issue has been fixed by disabling the failed conditional tasks and by adding skipped icons to them. This gives a better picture of the state of the pipeline run. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1880389[*BZ#1880389*])
+*OLM*
 
-* Previously, the pod scale up or down buttons were available for a single pod resource, and the page crashed when the user pressed the scale button. This issue has been fixed by not showing the scale up or down buttons for a single pod resource. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1909678[*BZ#1909678*])
+* The Marketplace Operator was written to report that the services it offered were degraded whenever the `marketplace-operator` pod exited gracefully, which would happen during routine cluster upgrades. This caused the pod to report as degraded during normal upgrades, which caused confusion. The Marketplace Operator no longer reports that it is degraded when it exits gracefully and is no longer flagged by Telemeter as degraded. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1838352[*BZ#1838352*])
```

Comment:
Maybe it's right, but "by Telemeter" doesn't sound right to me. I'd think it should either by "by telemetry" or "by the telemeter", but again I'm not positive.

Reply:
Sounds funny to me too. A quick search uncovered several uses of the `Telemeter client` in our docs. :thinking: . I think `the Telemeter client` would be better wording here.